src/templates/ja/2019/index.html

```diff
@@ -1,9 +1,9 @@
 {% extends "base/2019/index.html" %}
 
-{% block title %}The {{ year }} Web Almanac{% endblock %}
+{% block title %} {{ year }} Web Almanac{% endblock %}
```

Comment:
Nit, but ca we remove the unnecessary space?
```suggestion
{% block title %}{{ year }} Web Almanac{% endblock %}
```